### ISSUE TITLE
Outstanding CSS issues to close or fix once Tailwind conversion is complete 

### ISSUE THREAD

**Issue by ChargrilledChook** [TheOdinProject/theodinproject#3069]
There are a number of outstanding CSS issues that have been put on hold while the Tailwind conversion takes place. I'm collecting them here for greater visibility / easier ability to tick them off once the site has completely migrated to Tailwind. 

**CSS Todo**

- [x] #3068 
- [x] #3018
- [x] #2718
- [x] #2664 
- [x] #2644
- [x] #2587
- [x] #2540
- [x] #2537
- [x] #2538
- [x] #2532
- [x] #908

**Comment by ChargrilledChook** [comment]
Feel free to comment any other issues that belong here too and I'll add them to the list :memo:

**Comment by KevinMulhern** [comment]
This is super helpful, thanks for taking the time to do this @ChargrilledChook 🏅

**Comment by 1Popla** [comment]
odin exercise comment

**Comment by Lofty-Brambles** [comment]
Adding to this. Please check out this [Discord forum post](https://discord.com/channels/505093832157691914/1024715614062907573) for the issue regarding there not being an on-focus colour for the dark-mode for the lesson links in the course pages. @ChargrilledChook

**Comment by KevinMulhern** [comment]
Thanks for letting us know @Lofty-Brambles 💪
I replied on that forum post.

**Comment by ChargrilledChook** [comment]
Cheers @Lofty-Brambles, it's already on the list under #2587

**Comment by ChargrilledChook** [comment]
Closing since everything on this list is done :tada:
